Week-6/contigiousArray.js

```diff
@@ -0,0 +1,19 @@
+// time complexity - o(n) space complexity o(1)
+const maxSumSubArray = (nums = []) => {
```

Comment:
Can you please add the problem statement as well as comments to every question?

Reply:
added problem statements in each file